feat: optimize skywalking instancename (#4069)
apache/apisix: docs/en/latest/plugins/skywalking.md

| Name         | Type   | Default  | Description                                                          |
| ------------ | ------ | -------- | -------------------------------------------------------------------- |
| service_name | string | "APISIX" | service name for skywalking reporter                                 |

> | service_instance_name | string |"APISIX Instance Name" | service instance name for skywalking reporter，  set it to `$hostname` to get local hostname directly.|

Reviewer: ```suggestion
| service_instance_name | string |"APISIX Instance Name" | service instance name for skywalking reporter. If you want to use the machine's hostname directly, set it to `$hostname`.|
```

A little tweak for the doc
